cypress/integration/cypressActions.spec.js

```diff
@@ -1,11 +1,45 @@
 /// <reference types='cypress' />
 
-describe('Name of the group', () => {
+describe('User should be able', () => {
+  let user;
   before(() => {
-    
+    cy.visit('/');
+    cy.task('freshUser').then((freshUser) => {
+      user = freshUser;
+    });
+  });
+
+  it('to register successfully', () => {
+    cy.get('#firstName').type(user.username).should('have.value', user.username);
+    cy.get('#lastName').type(user.lastname).should('have.value', user.lastname);
+    cy.get('#userEmail').type(user.email).should('have.value', user.email);
+    cy.get('#genterWrapper > .col-md-9 > :nth-child(2) > .custom-control-label').click();
+    cy.get('#userNumber').type(user.phone).should('have.value', user.phone);
+    cy.get('#dateOfBirthInput').type('{selectall}').type(user.birthdate).should('have.value', user.birthdate);
+    cy.get('#subjectsInput').type(user.subject, { force: true }).should('have.value', user.subject);
+    cy.get('#react-select-2-option-0').click();
+    cy.get('#hobbiesWrapper > .col-md-9 > :nth-child(3) > .custom-control-label').click();
+    cy.get('#currentAddress').type(user.address).should('have.value', user.address);
+    cy.get('#state > .css-yk16xz-control > .css-1wy0on6 > .css-tlfecz-indicatorContainer > .css-19bqh2r').click();
+    cy.get('#react-select-3-option-0').click();
+    cy.get('#city > .css-yk16xz-control > .css-1hwfws3').click();
+    cy.get('#react-select-4-option-2').click();
+    cy.get('#submit').click();
+    cy.get('#example-modal-sizes-title-lg').should('contain', 'Thanks for submitting the form')
   });
 
-  it('', () => {
+  it('to register with data the same as entered', () => {
```

Comment:
again, this test starts where the previous one ended. "to register successfully" means that the user can register with the data they have entered, and the form after submitting shows this data to be validated. so it will just be a one rather extensive test